django-backend/fecfiler/transactions/managers.py

```diff
@@ -269,8 +269,10 @@ def get_queryset(self):
                 line_label=Case(
                     # Schedule A
                     When(_form_type="SA11A", then=Value("11(a)")),
-                    When(_form_type="SA11AI", then=Value("11(a)(i)")),
-                    When(_form_type="SA11AII", then=Value("11(a)(i)")),
+                    When(_form_type="SA11AI", itemized=True, then=Value("11(a)(i)")),
```

Comment:
i wonder if you moved the line_label clause to a later annotate, would you be able to use `form_type` rather than `_form_type`.  `form_type` should be mapped correctly based on itemization

Reply:
> i wonder if you moved the line_label clause to a later annotate, would you be able to use `form_type` rather than `_form_type`. `form_type` should be mapped correctly based on itemization

I tried to do this when first implementing the line_label case and it works, however it adds massively to the query size (since it generates large subqueries for calculating form_type for each case).  Also, the query performance degraded noticeably when I did this also.

Reply:
> I also wonder, we put `line_label` in the api because we needed to sort on it. Since we're sorting on `line_label_order_key` now, should the `line_label` mapping be defined on the front end now? I'm not sure.

I lean towards leaving it in the api since it is coupled to the line_label_order_key definition and adding a new label in app would mean still adding a line_label_order_key in api (which could more easily be forgotten I feel if they're not in the same place).  I am open if we want to move it however.